lib/components/Radio.js

```diff
@@ -11,18 +11,31 @@ const Radio = ({
   stacked = false,
   className = "",
   containerClassName = "",
+  error = "",
+  ...props
 }) => {
+  const id = useId(props.id);
+  const errorId = `error_${id}`;
   return (
     <div className={classnames(["neeto-ui-radio__wrapper", className])}>
-      {label && <Label>{label}</Label>}
+      {label && <Label className="neeto-ui-radio__label">{label}</Label>}
       <div
         className={classnames(["neeto-ui-radio__container"], {
           "neeto-ui-radio__container--stacked": stacked,
           [containerClassName]: containerClassName,
         })}
       >
-        {children}
+        {React.Children.map(children, (child) => React.cloneElement(child, {  ...child.props, error: !!error, ...props, value: child.props.value }))}
```

Comment:
@ajmaln _a, I am not clear on why this change is required.

Reply:
@edwinbbu This is to pass down the `onChange` and `onBlur` handlers (from formik which is given to the parent `Radio` component) to the `Radio.Item` component.

Reply:
@edwinbbu I just noticed a bug in this which causing initialValues to not being applied. Will add a fix for it.